lib/querystring.js

```diff
@@ -176,6 +177,9 @@ function encodeStringified(v, encode) {
     // escaping due to the inclusion of a '+' in the output
     return (MathAbs(v) < 1e21 ? '' + v : encode('' + v));
   }
+  if (typeof v === 'bigint' && NumberIsFinite(Number(v))) {
```

Comment:
Why the second condition?

Reply:
I added it because I saw the same in the above check. It seems unnecessary in this case, so I'll remove it.